src/Capgemini.PowerApps.PackageDeployerTemplate/PackageTemplateBase.cs

```diff
@@ -86,9 +91,16 @@ public override bool AfterPrimaryImport()
             this.DataImporterService.Import(this.ConfigDataStorage.DataImports?.Where(c => !c.ImportBeforeSolutions), this.PackageFolderPath);
             this.ProcessDeploymentService.Activate(this.ConfigDataStorage.ProcessesToActivate);
             this.WordTemplateImporterService.ImportWordTemplates(this.ConfigDataStorage.WordTemplates, this.PackageFolderPath);
+            this.flowConnectionService.ActivateFlows(this.ConfigDataStorage.Flows, _solutions);
 
             this.PackageLog.Log($"{nameof(PackageTemplateBase)}.{nameof(AfterPrimaryImport)} completed.", TraceEventType.Information);
             return true;
         }
+                
+        public override void PreSolutionImport(string solutionName, bool solutionOverwriteUnmanagedCustomizations, bool solutionPublishWorkflowsAndActivatePlugins, out bool overwriteUnmanagedCustomizations, out bool publishWorkflowsAndActivatePlugins)
```

Comment:
Something minor, but could this be moved to sit in-between `BeforeAnything` and `BeforeImportStage`. I'd like to the method ordering the same as they'd execute.

Reply:
Agree order makes sense